.github/workflows/publish-microzig.yml

```diff
@@ -0,0 +1,54 @@
+name: Deploy
+
+permissions:
+  contents: write
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
+      - "v*.*.*-*"
+
+jobs:
+  # Build the project to ensure it works. Also to get the binaries.
+  build-microzig:
+    uses: ./.github/workflows/build-base.yml
+    concurrency:
+      group: build
+      cancel-in-progress: true
+    with:
+      zig-version: ${{ vars.MZ_TARGET_ZIG_VERSION }}
+      get-submodules: true
+      is-packaged: true
+      target-os: macOS
+      github-artifact-name: microzig-build
+      artifact-output-path: boxzer-out
+      ref: ${{ github.ref_name }}
+    secrets:
+      downloads-url: ${{ secrets.BOXZER_DOWNLOADS_URL }}
+  # Alters the boxzer output to match what we desire
+  alter-microzig-artifact:
+    uses: ./.github/workflows/alter-microzig-artifact.yml
+    needs: build-microzig
+    with:
+      in-gh-artifact-name: microzig-build
+      out-gh-artifact-name: microzig-alt
+      zig-version: ${{ vars.MZ_TARGET_ZIG_VERSION }}
+      microzig-version: ${{ needs.build-microzig.outputs.version }}
+  # Publish microzig
+  publish-microzig:
+    uses: ./.github/workflows/publish-base.yml
+    needs: alter-microzig-artifact
+    concurrency:
+      group: publish
+      cancel-in-progress: false
+    with:
+      tag: ${{ github.ref_name }}
+      github-artifact-name: microzig-alt
+      source-path: /
+    secrets:
+      target-path: ${{ secrets.DEPLOY_MZ_ROOT_DATA_PATH }}
+      ssh-key: ${{ secrets.DEPLOY_MZ_PRIVATE_KEY }}
+      host: ${{ secrets.DEPLOY_MZ_HOST }}
+      port: ${{ secrets.DEPLOY_MZ_PORT }}
+      user: ${{ secrets.DEPLOY_MZ_USER }}
```

Comment:
Can you rename them to 
`DEPLOY_ROOT_DATA_PATH`
`DEPLOY_PRIVATE_KEY`
`DEPLOY_HOST`
`DEPLOY_PORT`
`DEPLOY_USER`
as otherwise i have to change them in the organizational level of ZEG

Reply:
Done with latest push